conf/general.yml-docker

```diff
@@ -95,7 +95,7 @@ PHOTO_STORAGE_OPTIONS:
                     # it doesn't already exist. Requires the appropriate AWS
                     # permissions.
 #  REGION: 'eu-west-1' # optional, only used if CREATE_BUCKET is set. Controls
-# which AWS region the S3 bucket will be created in.
+                       # which AWS region the S3 bucket will be created in.
```

Comment:
is there a reason for this?

Reply:
To match the conf/general.yml-example file, easier to diff :)